apps/src/p5lab/AnimationPicker/animation-picker-list-item.module.scss

```diff
@@ -0,0 +1,116 @@
+@import 'color.scss';
+
+$thumbnail-size: 105px;
+$thumbnail-border-width: 1px;
+$hover-plus-size: 24px;
+
+.root {
+    float: left;
```

Comment:
nit: styles are indented 1 tab too far (same thing about linting -- make a single change and see if our linting catches this so we don't have to do it manually anymore!)